### PR TITLE
docs(typescript): typos

### PR DESCRIPTION
**Pull request by Atinux**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Explaining how to activate stricter types should be more visible.

### CODE REVIEW

**Comment by pi0** [comment]
We could highlight and **explain** strict option better but I think the main `tsconfig` is a much more generic concept to remain above and explain as is an essential part of every nuxt project.

File: docs/content/2.guide/1.concepts/6.typescript.md [hunk 1]
```diff
@@ -10,6 +10,22 @@ By default, Nuxt doesn't check types when you run `nuxi dev` or `nuxi build`, fo
 yarn nuxi typecheck
 ```
 
+## Stricter Checks
```
Comment (DamianGlowala) [inline docs/content/2.guide/1.concepts/6.typescript.md:13]: Have we decided on whether we should keep section headlines lowercase (other than the first word, proper nouns, etc.)?
```suggestion
## Stricter checks
```

File: docs/content/2.guide/1.concepts/6.typescript.md [hunk 2]
```diff
@@ -10,6 +10,22 @@ By default, Nuxt doesn't check types when you run `nuxi dev` or `nuxi build`, fo
...
+
+TypeScript comes with certain checks to give you more safety and analysis of your program.
+
+Once you’ve converted your codebase to TypeScript and felt familiar with it, you can start enabling these checks for greater safety. ([read more](https://www.typescriptlang.org/docs/handbook/migrating-from-javascript.html#getting-stricter-checks))
```
Comment (DamianGlowala) [inline docs/content/2.guide/1.concepts/6.typescript.md:17]:
```suggestion
Once you’ve converted your codebase to TypeScript and felt familiar with it, you can start enabling these checks for greater safety ([read more](https://www.typescriptlang.org/docs/handbook/migrating-from-javascript.html#getting-stricter-checks)).
```

**A side note**
Not sure which library causes it (Docus vs Nuxt Content), but have any of you noticed there's always a whitespace appended to the links across th whole documentation?
image

File: docs/content/2.guide/1.concepts/6.typescript.md [hunk 3]
```diff
@@ -10,6 +10,22 @@ By default, Nuxt doesn't check types when you run `nuxi dev` or `nuxi build`, fo
...
+export default defineNuxtConfig({
+  typescript: {
+    strict: true
+  }
+})
+```
+
 ## Auto-generated types
 
 When you run `nuxi dev` or `nuxi build`, Nuxt generates the following files for IDE type support (and type-checking):
```
Comment (DamianGlowala) [inline docs/content/2.guide/1.concepts/6.typescript.md:31]:
```suggestion
When you run `nuxi dev` or `nuxi build`, Nuxt generates the following files for IDE type support (and type checking):
```

**Review by DamianGlowala — CHANGES_REQUESTED** [review]

File: docs/content/2.guide/1.concepts/6.typescript.md (hunk 2 above)
Comment (Atinux) [reply docs/content/2.guide/1.concepts/6.typescript.md:17]: This will be fixed with new version of DOcus @DamianGlowala

Comment (DamianGlowala) [reply docs/content/2.guide/1.concepts/6.typescript.md]: Great to hear, thanks for the info @Atinux!

**Comment by pi0** [comment]
Thanks ❤️

**Pull request merged**